presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkSuiteInfo.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.framework;
+
+import com.google.common.collect.ImmutableMap;
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public class BenchmarkSuiteInfo
+{
+    private final String suite;
+    private final String querySet;
+    private final List<PhaseSpecification> phases;
+    private final Map<String, String> sessionProperties;
+
+    @JdbiConstructor
+    public BenchmarkSuiteInfo(
+            @ColumnName("suite") String suite,
+            @ColumnName("query_set") String querySet,
+            @ColumnName("phases") List<PhaseSpecification> phases,
+            @ColumnName("session_properties") Optional<Map<String, String>> sessionProperties)
```

Comment:
`session_properties` is already required to be non null, so it does need to be optional